cluster/aws/util.sh

```diff
@@ -16,6 +16,19 @@
 
 # A library of helper functions and constant for the local config.
 
+# Experimental flags can be removed/renamed at any time.
+# The intent is to allow experimentation/advanced functionality before we
+# are ready to commit to supporting it.
+# Experimental functionality:
+#   KUBE_SHARE_MASTER=true
+#     Detects an existing master and reuse it; useful if you want to
+#     create more nodes, perhaps with a different instance type or in
+#     a different subnet/AZ
+#   KUBE_SUBNET_CIDR=172.20.1.0/24
```

Comment:
I was trying to provide a hint as to the typical usage - I imagine most people will not specify the default, but may then need a hint for the second AZ.  (Same as KUBE_SHARE_MASTER=true)